core/src/main/java/org/apache/carbondata/core/datastore/page/ColumnPage.java

```diff
@@ -530,6 +546,11 @@ private void putNull(int rowId) {
   public abstract byte[] getShortIntPage();
 
   /**
+   * Get boolean value page
+   */
+  public abstract byte[] getBooleanPage();
```

Comment:
The same as above, you can implement this function here instead of abstract

Reply:
Ok, I have changed it.